android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/SettingsFragment.java

```diff
@@ -46,37 +53,100 @@ public void onCreate(Bundle savedInstanceState) {
     getActivity().getActionBar().setDisplayHomeAsUpEnabled(true);
     // Load the preferences from an XML resource
     addPreferencesFromResource(R.xml.settings);
+    updatePwsList();
+  }
+
+  @Override
+  public void onResume() {
+    super.onResume();
+    getActivity().getActionBar().setTitle(R.string.title_settings);
```

Comment:
For consistency I think it should be done here at least for now. All the other fragments set the title in the `onResume` method.